angular/src/app/admin/dam-repository/resources/resource-form/resource-form.component.ts

```diff
@@ -73,6 +79,9 @@ export class ResourceFormComponent implements OnInit, AfterViewInit, Form {
         infoUrl: [dto.ui.infoUrl, [FormValidators.url]],
       }),
     });
+    this.form.valueChanges.pipe(
```

Comment:
I like it better named as https://github.com/DNAstack/ddap-frontend/pull/231/files#diff-ae5243b0ff08f9e76d36305744a78d29R94